src/Pages/AdminPage/Helpers/UserForm.js

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect, useState } from "react";
+import Parse from "parse";
+import "../AdminPage.css";
+
+function UserForm({ user, setUser, addUser }) {
+  const [chatRooms, setChatRooms] = useState([]);
+
+  useEffect(() => {
+    async function fetchChats() {
+      try {
+        const chatQuery = new Parse.Query("chat");
+        const chats = await chatQuery.find();
+
+        const chatData = chats.map((chat) => ({
+          id: chat.id,
+          chatNumber: chat.get("chat_number"),
+        }));
+        setChatRooms(chatData);
+      } catch (error) {
+        console.error("Error fetching chat rooms:", error);
+      }
+    }
+    fetchChats();
+    return () => {
+      setChatRooms([]);
+    };
+  }, []);
+
+  return (
+    <div className="forms">
+      <form
+        onSubmit={(e) => {
+          e.preventDefault();
+          addUser();
+        }}
+      >
+        <input
+          type="text"
+          name="userName"
+          placeholder="Username"
+          value={user.userName}
+          onChange={(e) => setUser({ ...user, userName: e.target.value })}
+          required
+        />
```

Comment:
Consider creating separate components for HTML elements such as `input`, `form`, `select`, `button`, etc.